src/components/brightness/BrightnessController.h

```diff
@@ -20,6 +20,9 @@ namespace Pinetime {
 
     private:
       Levels level = Levels::High;
+      uint16_t pwmSequence[1] = {10000};
```

Comment:
couldn't that just be a single value instead of an c-array?

Reply:
probably :sweat_smile: this was pulled from the previous PWM backlight PR and I'm not sure anyone here has figured out exactly how it's designed yet. Is on my todo list though